src/CopyrighFixer/CopyrighFixer/sign.h

```diff
@@ -0,0 +1,70 @@
+//#
+//# Copyright (C) 2021-2021 QuasarApp.
+//# Distributed under the lgplv3 software license, see the accompanying
+//# Everyone is permitted to copy and distribute verbatim copies
+//# of this license document, but changing it is not allowed.
+//#
+
+#ifndef SIGN_H
+#define SIGN_H
+
+#include "CopyrighFixer_global.h"
+#include "CopyrighFixer/owner.h"
+#include <QString>
+#include <QList>
+
+
+namespace CopyrighFixer {
+
+/**
+ * @brief The Signature class adds a copyright signature to each file based on the collected owner information.
+ */
+class CopyrighFixer_EXPORT Signature {
+public:
+    Signature();
+
+    /**
+     * @brief setlistOwners The method generates a list of owners.
+     * @param objOwner This is a structure with information about the owner.
+     */
+    void setlistOwners(Owner &objOwner);
```

Comment:
you must be use const reference to list of the owners
const QList<Owner>& val

If want to add method that append new owner into list then you must be create separate method for this